schema-bridge.graphql

```diff
@@ -690,10 +751,13 @@ type Pool @entity {
 
   " Pool hourly snapshots "
   hourlySnapshots: [PoolHourlySnapshot!]! @derivedFrom(field: "pool")
+
+  _lastDailySnapshotTimestamp: BigInt
+  _lastHourlySnapshotTimestamp: BigInt
 }
 
 type PoolRoute @entity {
-  " Bytes.fromUTF(({Pool ID}-{lowerChainID}-{greaterChainID}) "
+  " Bytes.fromUTF(({Pool ID}-{crossToken}-{lowerChainID}-{greaterChainID}) "
```

Comment:
I know 😅 
In this case, the chainIDs might have the issue of `12-23` being equal to `1-223`.

An alternative would be to always pad these numbers with a given number of zeroes or something. Since it is going to be abstracted by the library it’s fine. What do you think?

Reply:
Since we can do hyphens in the schema I think that is okay to keep in there. I like the look of it anyway